spec/unit/coinbase/address/external_address_spec.rb

```diff
@@ -74,6 +98,69 @@
     end
   end
 
+  shared_examples 'it builds a staking operation' do |operation|
+    let(:transaction) { instance_double('Coinbase::Client::Transaction') }
+    let(:staking_operation) do
+      instance_double(
+        'Coinbase::Client::StakingOperation',
+        transactions: [transaction]
+      )
+    end
+
+    before(:each) do
```

Comment:
**nit**: I believe `before(:each)` is the default behavior, and we can just use `before`

Reply:
old habits die hard :P Will update!